core-types/ints_and_floats_test.rb

```diff
@@ -2,24 +2,23 @@
 gem 'minitest', '~> 5.2'
 require 'minitest/autorun'
 require 'minitest/pride'
-require 'minitest/homework'
-
-class FixnumTest < Minitest::Homework
-  be_gentle!
 
+class FixnumTest < Minitest::Test
   def test_add
     assert_equal __, 6.send(:+, 3)
```

Comment:
I'm not super stoked about showing them send here. I think it mostly would be confusing at this point. Thoughts?